_includes/1.0/overview/security/oauth2/refresh.html

```diff
@@ -0,0 +1,81 @@
+<div id="refresh" class="group">
+
+    <h2>Refresh the Access Token</h2>
+
+    <p>The <code>refresh_token</code> received when exchanging the authorization code for an access token can be used to exchange for a new access token.</p>
+
+    <p>This is particularly useful in the case of the original access token has expired and the application does not want to start a new authorization flow.</p>
+
+    <p>If you make a request to OneUp API and the token has expired already, you’ll get back a response indicating as such. You can check for this specific error message, and then refresh the token and try the request again.</p>
```

Comment:
Can you add an example and the HTTP code of the response?

Reply:
Added with the commit [908f6b3724d13700528479bf0e305807c8ca9161](https://github.com/myERP/myerp.github.io/pull/31/commits/908f6b3724d13700528479bf0e305807c8ca9161)